test/metabase/db/schema_migrations_test.clj

```diff
@@ -965,9 +965,12 @@
                new-paths-set))))))
 
 (deftest migrate-field-database-type-test
-  (testing "Migration v47.00-003: set base-type to type/JSON for JSON database-types for postgres and mysql"
-    (impl/test-migrations ["v47.00-001"] [migrate!]
-      (let [[pg-db-id
+  (testing "Migration v47.00-001: set base-type to type/JSON for JSON database-types for postgres and mysql"
+    (impl/test-migrations ["v47.00-001"] [_]
+      (let [{:keys [db-type ^javax.sql.DataSource
+                    data-source]} mdb.connection/*application-db*
+            migrate! (partial db.setup/migrate! db-type data-source)
```

Comment:
I'd have a comment here to why we use this

Reply:
Done https://github.com/metabase/metabase/commit/3c14be2992a1c42fddf251aac26b53d87af329b5